app/components/forms/orders/order-form.js

```diff
@@ -202,7 +202,8 @@ export default Component.extend(FormMixin, {
     };
 
     let companyValidation = {
-      rules: [
+      identifier : 'company',
+      rules : [
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing space after key 'rules'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5182068)

Reply:
added

Reply:
Conversations are resolved automatically. Please don't resolve the conversations yourself. The fact that it is not resolved automatically means that the issue is not resolved